examples/waiter/waiter.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	"github.com/stackitcloud/stackit-sdk-go/core/utils"
 	"github.com/stackitcloud/stackit-sdk-go/services/dns"
+	"github.com/stackitcloud/stackit-sdk-go/services/dns/wait"
```

Comment:
I would go with the alias `dnsWait` to make it more clear, since we also have the `core/wait` pkg

Reply:
I'd leave it like this. It's short and clear, looking at the code you'll easily assume that it's the wait for the dns, and it's unlikely conflict with other wait packages will happen: I don't think you ever want to import `core/wait`, and if you end up using waiters from multiple services, one can then use `[SERVICE]wait` for an alias

(Also small note, package names should be lower case be convention: https://go.dev/blog/package-names, so I'd keep the alias lower case as well)

Reply:
Had to remove waiter example, the lint was complaining that the dns release didn't have the `wait` package